devdocs/process.md

```diff
@@ -146,7 +155,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
 ###Reviewing a fix
 Role: reviewer
 
-  * This is a code quality review. No need to run tests.
+  * Ensure that the Travis CI build is successful and the developer has local dev green.
```

Comment:
Does this mean that we now have to ensure _two_ kinds of green instead of one? It seems like Travis is adding an overhead here.